frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
I find the switch more readable and it is also consistant with `view/statechart.js`. It may also be faster (benchmark needed) and prevent the creation of a temporary array (which may be cached, but which will make thing again less readable).